converters/fromZigbee.js

```diff
@@ -5313,6 +5313,33 @@ const converters = {
             }
         },
     },
+    ZGRC_KEY_009_cmdMoveHue: {
+        cluster: 'lightingColorCtrl',
+        type: 'commandMoveHue',
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.movemode === 0) {
+                return {action: 'circle_release'};
+            } else {
+                return {action: 'circle_hold'};
+            }
+        },
+    },
+
+    ZGRC_KEY_009_cmdMoveColorTemp: {
+        cluster: 'lightingColorCtrl',
+        type: 'commandMoveColorTemp',
+        convert: (model, msg, publish, options, meta) => {
+            return {action: 'W_hold'};
+        },
+    },
+
+    ZGRC_KEY_009_cmdMoveColorTemp_release: {
```

Comment:
Could you provide the herdsman debug logging of this  command?

To enable herdsman debug logging, see https://www.zigbee2mqtt.io/information/debug.html#zigbee-herdsman-debug-logging